src/main/resources/assets/computercraft/lua/rom/programs/copy.lua

```diff
@@ -10,7 +10,9 @@ local sDest = shell.resolve( tArgs[2] )
 local tFiles = fs.find( sSource )
 if #tFiles > 0 then
     for n,sFile in ipairs( tFiles ) do
-        if fs.isDir( sDest ) then
+        if fs.exists( sDest ) == true then
```

Comment:
The two styles produce a different result in bytecode[1], but have basically the same effect.

[1]:
```
viluon@deluon:~$ luac5.1 -l -o /dev/null -
local x

if x then end

if x == true then end

main <stdin:0,0> (5 instructions, 20 bytes at 0xc98230)
0+ params, 2 slots, 0 upvalues, 1 local, 1 constant, 0 functions
	1	[3]	TEST     	0 0 0
	2	[3]	JMP      	0	; to 3
	3	[5]	EQ       	0 0 -1	; - true
	4	[5]	JMP      	0	; to 5
	5	[5]	RETURN   	0 1
```